src/config/validators/index.ts

```diff
@@ -5,26 +5,19 @@
  * Use upper camel-case for your SVG import, lower camel case for the svg.
  * import { ReactComponent as ValidatorEntityName } from './thumbnails/validatorEntityName.svg';
  */
-import { ReactComponent as Cere } from './thumbnails/cere.svg';
```

Comment:
Please, don't replace Cere. Add "AnyValid" next to Cere. You can use this [PR](https://github.com/Cerebellum-Network/staking-dashboard/pull/24) as an example.